File: java/Ram.java
class Ram{
	// atribut private
    private String capacity;
	private String price;

	// constructor
	public Ram(){}
	public Ram(String capacity, String price){
		this.capacity = capacity;
		this.price = price;
	}

	// setter/getter
	public void setcapacity(String capacity) {
		this.capacity = capacity;
	}
	public String getcapacity() {
		return capacity;
	}
	public void setprice(String price) {
		this.price = price;
	}
	public String getprice() {
		return price;
	}

	/*Method*/
	void printRam(){
		System.out.println("Ram capacity : " + this.capacity);
		System.out.println("Ram price : " + this.price);
	}
}

File: php/Ram.php
<?php

class Ram{
	// atribut private
    private $capacity;
    private $price;

	// constructor
	public function __construct($capacity, $price){
        $this->capacity = $capacity;
        $this->price = $price;
	}

	// setter/getter
    public function setcapacity($capacity){
		$this->capacity = $capacity;
	}
    public function getcapacity(){
		return $this->capacity;
	}

    public function setprice($price){
		$this->price = $price;
	}
    public function getprice(){
		return $this->price;
	}

	/*Method*/
	public function printRam(){
		echo "Ram capacity : ".$this->capacity."<br>";
		echo "Ram price : ".$this->price."<br>";
	}
}

?>